src/index.js

```diff
@@ -9,12 +9,24 @@ const todoList = document.getElementById('todoList');
 
 function addItem() {
 	if (userInput.value === '') {
-		alert('You really got nothing to do?')
+		alert('You really got nothing to do?');
 	}
 	else {
 		const li = document.createElement('li');
 		li.innerHTML = userInput.value;
-		todoList.appendChild(li)
+		todoList.appendChild(li);
+		let span = document.createElement('span');
+		span.innerHTML = '\u00d7';
+		li.appendChild(span);
 		userInput.value = ''
 	}
 }
+
+todoList.addEventListener('click', function (e) {
+	if (e.target.tagName === 'LI') {
```

Comment:
Here you could use a `switch`, `case`: https://www.w3schools.com/js/js_switch.asp

Reply:
You should not check by tag name, you should add an event in each element

Reply:
yeah im doing that tomorrow